x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/setup_guide.tsx

```diff
@@ -32,15 +32,15 @@ import './setup_guide.scss';
  * customizable, but the basic layout and instruction steps are DRYed out
  */
 
-interface SetupGuideProps {
+interface Props {
   children: React.ReactNode;
   productName: string;
   productEuiIcon: 'logoAppSearch' | 'logoWorkplaceSearch' | 'logoEnterpriseSearch';
   standardAuthLink?: string;
   elasticsearchNativeAuthLink?: string;
 }
 
-export const SetupGuide: React.FC<SetupGuideProps> = ({
+export const SetupGuideLayout: React.FC<Props> = ({
```

Comment:
Jason previously left this feedback that it was confusing to rename `SetupGuide` to `SetupGuideLayout` in other setup guide component/views, so I officially renamed it here. It also makes a lot more sense now that the instructions are in a separate component!